simplefi-1inch/src/Mooniswap.ts

```diff
@@ -0,0 +1,292 @@
+import { Address, BigInt, ethereum, store } from "@graphprotocol/graph-ts"
+
+import {
+  Account as AccountEntity,
+  AccountLiquidity as AccountLiquidityEntity,
+  Burn as BurnEntity,
+  Market as MarketEntity,
+  Mint as MintEntity,
+  Pair as PairEntity
+} from "../generated/schema"
+
+import {
+  Sync,
+  Deposited,
+  Withdrawn,
+  Transfer,
+} from "../generated/templates/Mooniswap/Mooniswap"
+
+import {
+  ADDRESS_ZERO,
+  getOrCreateAccount,
+  investInMarket,
+  redeemFromMarket,
+  TokenBalance,
+  updateMarket
+} from "./common"
+
+const BASE_SUPPLY = BigInt.fromI32(1000)
+
+function getOrCreateMint(event: ethereum.Event, pair: PairEntity): MintEntity {
+  let mintId = pair.id.concat("-").concat(event.transaction.hash.toHexString())
+  let mint = MintEntity.load(mintId)
+  if (mint != null) {
+    return mint as MintEntity
+  }
+
+  mint = new MintEntity(mintId)
+  mint.pair = pair.id
+  mint.transferEventApplied = false
+  mint.syncEventApplied = false
+  mint.mintEventApplied = false
```

Comment:
These are not used, right? Can we remove those params from schema?

Reply:
Oh yeh, forgot to remove those!